tests/functional/settings_clients.js

```diff
@@ -177,46 +157,31 @@ define([
 
         // test cancel
         .then(click('.cancel-disconnect'))
-
-        .then(pollUntilGoneByQSA('#client-disconnect'))
+        .waitForDeletedByCssSelector('#client-disconnect')
```

Comment:
This is the important change, instead of `pollUntilGoneByQSA`, use `waitForDeletedByCssSelector` which seems to cause an update of the element cache.